src/oneid/oneid-lambda-client-registration/src/main/resources/application.properties

```
@@ -37,6 +37,6 @@ sns_topic_notification_environment=${SNS_TOPIC_NOTIFICATION_ENVIRONMENT:d}
 quarkus.native.additional-build-args=--initialize-at-run-time=com.nimbusds.oauth2.sdk.id.Identifier
 # cors config
 quarkus.http.cors=true
-quarkus.http.cors.origins=http://localhost
+quarkus.http.cors.origins=${QUARKUS_HTTP_CORS_ORIGINS:d}
```

according to [docs](https://quarkus.io/guides/config-reference) this is not needed, env always override what is written in `application.properties`. I suggest to keep the original configuration:
```suggestion
quarkus.http.cors.origins=http://localhost
```
which could help in local development situation